Automattic/woocommerce-payments
# Expose WCPay refund transaction id for refund endpoint

Fixes #7857 

#### Changes proposed in this Pull Request
Expose refund transaction id for the refund API endpoint. The transaction id is saved on the refund order in the metadata, with `_refund_transaction_id ` key.









#### Testing instructions





* Create REST API Keys as described [here](https://woocommerce.github.io/woocommerce-rest-api-docs/#rest-api-keys)
* create a refund via via https://woocommerce.github.io/woocommerce-rest-api-docs/#create-a-refund . 
* check the api response, the metadata should contain `_refund_transaction_id`
* check the transactions in `Payments -> Transactions`

https://github.com/Automattic/woocommerce-payments/assets/82826872/7ec9798c-6149-4dae-92be-f9e1039abd62



-------------------


**Post merge**
